### PR TITLE
chore(zones): adds more content hooks for zone creation

### PR DESCRIPTION
**Pull request by kleinfreund**
**chore: adds text gradient utility**

Adds a text-gradient utility class and uses it for the Zone creation title.

**chore(zones): adds content hooks for descriptions and facts to creation view**

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

File: src/app/zones/views/ZoneCreateView.vue [hunk 1]
```diff
@@ -41,17 +41,45 @@
...
-                <h1 class="form-title">
-                  {{ t('zones.routes.create.title') }}
-                </h1>
+              <div class="form-header">
+                <div>
+                  <h1 class="form-title">
+                    <span class="text-gradient">{{ t('zones.form.title') }}</span>
+                  </h1>
+
+                  <p v-if="t('zones.form.description') !== ' '">
```
Comment (johncowen) [inline src/app/zones/views/ZoneCreateView.vue:50]: Would `''` and a length check work here?

File: src/app/zones/views/ZoneCreateView.vue [hunk 2]
```diff
@@ -41,17 +41,45 @@
...
+                    class="fact-list__item"
+                  >
+                    <KIcon
+                      icon="check"
+                      :color="KUI_COLOR_TEXT_SUCCESS"
+                    />
+
+                    {{ fact }}
+                  </div>
+                </div>
```
Comment (johncowen) [inline src/app/zones/views/ZoneCreateView.vue:71]: Bit of a nit but should we use a `ul/li` here instead of `div`s?

File: src/app/zones/views/ZoneCreateView.vue [hunk 3]
```diff
@@ -234,6 +270,10 @@
                     <h2 class="form-section-title">
                       {{ t('zones.form.section.scanner.title') }}
                     </h2>
```
Comment (johncowen) [inline src/app/zones/views/ZoneCreateView.vue:272]: I know its not part of this PR but I noticed that when the zone has been found, it looks really strange that it still says "Waiting for Zone to be connected...", should we just remove this header?

**Review by johncowen — COMMENTED** [review]
Left a few inlines

File: src/app/zones/views/ZoneCreateView.vue (hunk 3 above)
Comment (kleinfreund) [reply src/app/zones/views/ZoneCreateView.vue:272]: It’s better than not arbitrary having a form section without a header. It’s also unambiguous that the Zone is connected when the widget says so so I don’t see this as a conflict.

File: src/app/zones/views/ZoneCreateView.vue (hunk 1 above)
Comment (kleinfreund) [reply src/app/zones/views/ZoneCreateView.vue:50]: Yep, still the case:

```
Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "zones.form.section.name.description" for locale "en-us", using id as fallback.
```

File: src/app/zones/views/ZoneCreateView.vue (hunk 3 above)
Comment (kleinfreund) [reply src/app/zones/views/ZoneCreateView.vue:271]: It doesn’t look broken to me, no.

It doesn’t make sense to change the wording of a section heading conditionally to some state inside the section.

I’m happy to change the title to something not related to the state, but I’m not removing it because that’s inconsistent with the form layout.

File: src/app/zones/views/ZoneCreateView.vue (hunk 2 above)
Comment (kleinfreund) [reply src/app/zones/views/ZoneCreateView.vue:70]: Changed this.

File: src/app/zones/views/ZoneCreateView.vue (hunk 1 above)
Comment (johncowen) [reply src/app/zones/views/ZoneCreateView.vue:50]: Hmm thats weird we do an `undefined` check for this, oh this one is coming directly from formatjs.

Umm ok. I'd like to be able to specify `''` when we need to if possible. I'd actually like to use the error config that was added to `@kong-ui-public/i18n` a while back, oh and actually a couple of other things now come to think of it. I'll make an issue

Thanks for trying it out tho!

File: src/app/zones/views/ZoneCreateView.vue (hunk 3 above)
Comment (johncowen) [reply src/app/zones/views/ZoneCreateView.vue:271]: Ok maybe lets bring it up with @johnharris85 ? Feel free to resolve for now, we can add an issue for this if folks want to tweak.

File: src/app/zones/locales/en-us/index.yaml [hunk 4]
```diff
@@ -74,15 +74,22 @@ zones:
     create: 'Create Zone'
   form:
     exit: 'Exit'
+    title: 'Create & connect Zone'
+    description: ' '
+    facts: ['One', 'Two', 'Three', 'Four']
```
Comment (johncowen) [inline src/app/zones/locales/en-us/index.yaml:79]: Not sure if you meant to leave these in or not, will approve anyway, but just incase you didn't notice these

**Review by johncowen — APPROVED** [review]
LGTM!

File: src/app/zones/locales/en-us/index.yaml (hunk 4 above)
Comment (kleinfreund) [reply src/app/zones/locales/en-us/index.yaml]: Thanks, removed those.
